pandas/tests/frame/test_operators.py

```diff
@@ -218,6 +218,20 @@ def test_logical_with_nas(self):
         expected = Series([True, True])
         tm.assert_series_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "A, B, C",
```

Comment:
Can you name these `"left, right, expected"`?

Reply:
Thanks, @mroeschke - done.